content/blog/2020/04/2020-04-28-jenkins-is-the-way.adoc

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "Call for User Stories - Jenkins is the Way"
+tags:
+- announcement
+author: alyssat
+opengraph:
+  image: /images/post-images/jenkins-is-the-way/jenkins-is-the-way-opengraph.png
+---
+
+image:/images/post-images/jenkins-is-the-way/jenkins-is-the-way.png[Jenkins Is The Way, role=center, float=right, height=284]
```

Comment:
Would it make sense to add this logo to the artwork section from the very beginning? 
I'd guess we are going to reuse it somewhere else soon, e.g. on https://www.jenkins.io/sigs/advocacy-and-outreach/outreach-programs/ or https://www.jenkins.io/artwork/

Reply:
Logo has been added in 2a2ddacb38b02f4967c3650b43ec65d9bb7a8ca3 .  The edge of the logo image is not as well-defined at the transition to transparency as other logos.  I would need help from others with more image editing experience to correct that.